src/main/java/com/radixdlt/crypto/DefaultSignatures.java

```diff
@@ -0,0 +1,29 @@
+package com.radixdlt.crypto;
+
+public abstract class DefaultSignatures {
```

Comment:
Not quite clear to me why we should have an abstract class for "DefaultSignatures". Is this supposed to be an interface? A utility class?

Reply:
It is not abstract anymore, but not constructable. It was a suggestion by @clstrfsck to move out some `default*` methods from the interface `Signatures`.